CefSharp.BrowserSubprocess/Program.cs

```diff
@@ -19,16 +18,16 @@ public static int Main(string[] args)
             SubProcess.EnableHighDPISupport();
 
             int result;
+            var type = args.GetArgumentValue(CefSharpArguments.SubProcessTypeArgument);
+            var parentProcessId = int.Parse(args.GetArgumentValue(CefSharpArguments.HostProcessIdArgument));
 
-            const string typePrefix = "--type=";
-            var typeArgument = args.SingleOrDefault(arg => arg.StartsWith(typePrefix));
-            var type = typeArgument.Substring(typePrefix.Length);
+            Task.Run(() => AwaitParentProcessExit(parentProcessId));
```

Comment:
Whilst the `ThreadPool` won't be used in the render process, it just doesn't seem appropriate to use a vanilla `Task.Run`, should probably add `TaskCreationOptions.LongRunning`, from memory it should spawn a new `Thread` and a quick check on `stackoverflow` suggests it will.

https://stackoverflow.com/a/10298761

Reply:
I though about adding that, but since this process is running mostly unmanaged code and is not using the ThreadPool, I skipped it. But its a small change, I will do it.